DigitalOffice/Services/CheckRightsService/Commands/Interfaces/IAddRightsForUserCommand.cs

```diff
@@ -0,0 +1,17 @@
+﻿using LT.DigitalOffice.CheckRightsService.RestRequests;
+
+namespace LT.DigitalOffice.CheckRightsService.Commands.Interfaces
+{
+    /// <summary>
+    /// Add rights for user
```

Comment:
We write comments with a dot at the end.

Reply:
fixed